doc/src/input/mc.md

```diff
@@ -24,32 +24,42 @@ center of mass.
 Currently, all Monte-Carlo simulations are carried out using Metropolis 
 acceptance criteria.
 
+You can add all necessary information after the `[simulations.propagator]` label.
+
+- Needed keys:
+    * `type = "MonteCarlo"`
+    * `temperature` (string): System temperature. The string contains the 
+temperature with unit.
+
+- Optional keys:
+    * `update_delta_every` (uint): After this number of steps of a move, `delta` values 
```

Comment:
`unsigned integer` or `positive integer` or just `integer` might be more readable than `uint` here.